__tests__/topic.tests.js

```diff
@@ -105,3 +105,32 @@ describe("GET /api/articles/:articles_id", () => {
       });
   });
 });
+describe("GET /api/articles", () => {
+  test("Responds with an array of all articles object", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body.articles;
+        expect(Array.isArray(articles)).toBe(true);
+      });
+  });
```

Comment:
This can be included on another test, it's not so substantial as to demand it's own test block